pages/faq.tsx

```diff
@@ -115,7 +115,7 @@ export default function FaqPage(props: FaqPageProps) {
                       <p className="text-base text-gray-500">
                         {htmr(question.jawaban)}
                       </p>
-                      <small>
+                      <small className="mt-2 inline-block">
```

Comment:
```suggestion
                      <small className="mt-4 inline-block">
```
If `mt-2` is too little, let's try `mt-4`, then.

Reply:
updated to `mt-4`
<img width="586" alt="image" src="https://user-images.githubusercontent.com/19145812/127741605-281647dd-ee21-4ad1-bec7-435d602ce895.png">